alchemy/test/cloudflare/secret.test.ts

```diff
@@ -12,123 +12,131 @@ const test = alchemy.test(import.meta, {
   prefix: BRANCH_PREFIX,
 });
 
-// skip because Cloudflare only allows one Secret Store per account ...
-describe.skip("Secret Resource", () => {
-  const testId = `${BRANCH_PREFIX}-test-secret`;
-  const storeId = `${BRANCH_PREFIX}-test-secret-store`;
-
+describe("Secret Resource", () => {
   test("create and delete secret in store", async (scope) => {
+    // Generate random secret name to avoid conflicts in parallel test execution
+    const secretName = `${BRANCH_PREFIX}-test-secret-${Math.floor(Math.random() * 10000)}`;
```

Comment:
Secret names should be deterministic and unique to all tests. This can leave dangling state on each test run.